rust.hcl

```diff
@@ -10,12 +10,6 @@ linux {
   source = "https://static.rust-lang.org/dist/rust-${version}-x86_64-unknown-linux-musl.tar.xz"
 }
 
-version "1.51.0" "1.52.1" "1.53.0" "1.54.0" "1.55.0" {
```

Comment:
Is this change in this PR on purpose?

Reply:
Semi on purpose :) - I needed to change it to test the CI, but got a little too exuberant.